quantum-core/lib/api.js

```diff
@@ -71,6 +71,7 @@ function buildPage (sourceFile, pipeline, config, logger, addLiveReload) {
   const start = Date.now()
   return Promise.resolve(pipeline(sourceFile))
     .then(files => Array.isArray(files) ? files : [files])
+    .then(files => Array.from(new Map(files.map(f => [f.info.dest, f])).values()))
```

Comment:
This change de-duplicates the writing of static files (e.g. `quantum-docs.css`) - this was particularly an issue for versioned pages as we had 47 copies of 8 files being written which caused a noticeable impact when building the docs